cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/AppMetadataStore.java

```diff
@@ -269,18 +282,25 @@ public void scanApplications(Cursor cursor,
         StoreDefinition.AppMetadataStore.APPLICATION_DATA_FIELD);
   }
 
-  public long getApplicationCount() throws IOException {
+  private Collection<Range> exceptSystemNamespace() {
```

Comment:
This is not a very good method name. What does it supposed to do? get ranges that excludes system namespace? If that's the case, call it `getNonSystemNamespaceRanges`